src/Positioner/Inline.php

```diff
@@ -29,7 +29,10 @@ function position(AbstractFrameDecorator $frame): void
         $block = $frame->find_block_parent();
 
         if (!$block) {
-            throw new Exception("No block-level parent found.  Not good.");
+            // FIXME: An inline frame without block parent should not be
+            // possible, but this can occur currently when the body is styled
+            // with `display: inline !important;`
+            return;
```

Comment:
FYI I was unable to get that exception thrown with the body styled as inline. Things just generally fell apart for positioning child inline elements because the positioner was never called from what I could see. But also I was testing with a fairly simple document.

I did run across the exception when the body was styled inline-block. In that case because it's a block type you do still have line boxes. The render wasn't significantly different from just `return`ing except that the containing block appears to be taken into account.

I'm fine moving forward with just a return for now since this is an edge case.